composer.json

```diff
@@ -48,5 +48,6 @@
         "branch-alias": {
             "dev-master": "3.2-dev"
         }
-    }
+    },
```

Comment:
like previously noted, this should go in travis configuration

Reply:
Fixed